src/secure_mount.rs

```diff
@@ -113,17 +113,26 @@ fn mount() -> Result<String, i32> {
         }
 
         // mount tmpfs with secure directory
-        tpm::run(
+        match tpm::run(
```

Comment:
Pull this tpm::run result into a temp variable, then use an `if let` with early exit for the error.

Reply:
Ok.